Translation corrections (#446)
07th-mod/higurashi-console-arcs: Update/ura_013.txt

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#f5e6d3>前野</color>", NULL, "<color=#f5e6d3>Maeno</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 0, "ps3/s14/00/vt4bb_maeno1131", 256, TRUE);
	OutputLine(NULL, "「それにしても…塚田も大したタマだな」",

> 			NULL, "\"Still... this Tsukada guy is quite a guy.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "\"Still... this Tsukada guy has balls of steel.\"", Line_Normal);
```

	ClearMessage();

//前野rvS14/00/VT4Bb_maeno1132.「学生時代は非常に打算的、付き合う相手も自分にとって有用かどうかで決めるような性格だった…とは当時の知人たちの評価だが、kvS14/00/VT4Bb_maeno1133.…一人の惚れた相手のために全てを投げ捨てて戦う、か」
	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#f5e6d3>前野</color>", NULL, "<color=#f5e6d3>Maeno</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 0, "ps3/s14/00/vt4bb_maeno1132", 256, TRUE);
	OutputLine(NULL, "「学生時代は非常に打算的、付き合う相手も自分にとって有用かどうかで決めるような性格だった…とは当時の知人たちの評価だが、",

> 			NULL, "\"Back in college, his acquaintances' evaluation of him... was that he had a very calculating personality, deciding who to date based on how much use they had to him. ", GetGlobalFlag(GLinemodeSp));

Reviewer: ```suggestion
			NULL, "\"Back in college, his acquaintances said... he had a very calculating personality, deciding who to date based on how much use they had to him. ", GetGlobalFlag(GLinemodeSp));
```
